internal/constants.go

```diff
@@ -0,0 +1,52 @@
+package internal
+
+import (
+	"fmt"
+	submariner "github.com/submariner-io/submariner-operator/api/submariner/v1alpha1"
+	"github.com/submariner-io/submariner-operator/pkg/images"
+	"github.com/submariner-io/submariner-operator/pkg/names"
+	"github.com/submariner-io/submariner-operator/pkg/subctl/cmd/utils"
+	"strings"
+)
+
+const (
+	OperatorNamespace = "submariner-operator"
+)
+
+func OperatorImage(imageVersion string, repo string, imageOverrideArr []string) string {
+
+	if imageVersion == "" {
+		imageVersion = submariner.DefaultSubmarinerOperatorVersion
+	}
+
+	if repo == "" {
+		repo = submariner.DefaultRepo
+	}
+
+	return images.GetImagePath(repo, imageVersion, names.OperatorImage, names.OperatorComponent, GetImageOverrides(imageOverrideArr))
+}
+
+func GetImageOverrides(imageOverrideArr []string) map[string]string {
```

Comment:
Given the above suggestion, this function would become:

```suggestion
func GetOverrides(imageOverrideArr []string) map[string]string {
```